src/js/common/globalOrLiteral.js

```diff
@@ -0,0 +1,27 @@
+/* global require */
+(function (fluid) {
+    "use strict";
+    fluid = fluid || require("fluid");
+    var gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.express.user");
+
+    /**
+     *
+     * Evaluate a single entry and retrieve the associated global variable if possible.
```

Comment:
Be a bit clearer about the workflow here, and what object is returned, since the logic is a little complex. If the argument is a string, it is looked up, and any value other than undefined is returned, else {} .... otherwise the input argument is returned. Does this really need to be this complex? I don't see that it is used elsewhere in the pull.
In my experience, functions like this which include lots of special cases in order to be "helpful" end up causing problems down the line.

Reply:
I think it was left over from a previous approach, I have deleted it.